db/version_set.cc

```diff
@@ -1812,7 +1812,7 @@ void Version::MultiGet(const ReadOptions& read_options, MultiGetRange* range,
         user_comparator(), merge_operator_, info_log_, db_statistics_,
         iter->s->ok() ? GetContext::kNotFound : GetContext::kMerge, iter->ukey,
         iter->value, nullptr, &(iter->merge_context), true,
-        &iter->max_covering_tombstone_seq, this->env_, &iter->seq,
+        &iter->max_covering_tombstone_seq, this->env_, nullptr,
```

Comment:
Does this mean we no longer support retrieving the sequence numbers for MultiGet?

Reply:
It was never actually supported. As mentioned in a previous comment, we may have to revisit when we support MultiGetForUpdate().